propel/resource_metric.go

```diff
@@ -467,7 +491,7 @@ func expandMetricFilters(def []interface{}) ([]*pc.FilterInput, diag.Diagnostics
 		f := &pc.FilterInput{
 			Column:   filter["column"].(string),
 			Operator: pc.FilterOperator(filter["operator"].(string)),
-			Value:    filter["value"].(string),
+			Value:    filter["value"].(*string),
```

Comment:
Regenerating the client changed this, I needed to adapt the tests also. Can this give problems?

Reply:
yes, see the next comment